### ISSUE TITLE
LENOVO V15 Gen4 AMD Touchpad & Keyboard not working

### ISSUE THREAD

**Issue by susanne99** [linuxmint/Cinnamon#12043]
### Distribution

Mint 21.3 

### Package version

6.0.4

### Frequency

Always

### Bug description

I bought a Lenovo V15 Gen 4 AMD notebook. Unfortunately, Linux Mint 21.3 doesn't work there. Keyboard and touchpad are dead. I tried all i8042 settings, and finally I installed the released kernel "6.5.0-14 generic" via the GUI. I read that it has already been fixed, but it still doesn't work for me. What can I do ? I urgently need help, I actually wanted to leave Microsoft

### Steps to reproduce

Fresh Installation of Linux Mint 21.3 on a Lenovo V15 Gen4 AMD Notebook

### Expected behavior

Built in Keyboard and Touchboard have to work

**Comment by leigh123linux** [comment]
Report the problem at https://github.com/linuxmint/linuxmint/issues

**Comment by fredcw** [comment]
Yes, it looks like it was fixed in kernel 6.5.10 and later https://bugzilla.kernel.org/show_bug.cgi?id=218003 so don't know why it isn't working.

Tbh, I'm not sure what deb package version numbers mean. Does "6.5.0-14 generic" mean kernel version 6.5.0 or version 6.5.14? Maybe someone else knows?

**Comment by fredcw** [comment]
You could try installing kernel 6.6 https://9to5linux.com/you-can-now-install-linux-kernel-6-6-on-ubuntu-heres-how

**Comment by susanne99** [comment]
ok, i will try the latest 6.7.1 with "Mainline Kernels" Tool

**Comment by susanne99** [comment]
the only way that works on lenovo v15 gen4 amd notebook is via 
![kernelinstalllation](https://github.com/linuxmint/cinnamon/assets/112922199/e4a24ff6-ff2d-410d-b422-cdebf7b35761)

**Comment by fredcw** [comment]
It seems it would be difficult to get a newer kernel on Mint atm. I would suggest trying Manjaro Cinnamon. You can easily install kernel versions 6.6 & 6.7 after you've installed Manjaro by going to System Settings>Manjaro Settings Manager>Kernel

**Comment by susanne99** [comment]
ok, thank you four the tip.
I read on the internet that manjaro isn't supposed to be that stable, so stay away from it, right?

**Comment by fredcw** [comment]
I'm not at my computer rn, so can't check if Ubuntu has a newer kernel, but I suspect it doesn't. Manjaro is fine as long as you do 2 things, before each update, every 2-3weeks, just check the update notes on the website to see if there any manual intervention you have do like edit some config file, but that's very rare. I think I had to edit some file about twice in 3 years. The other thing is just to make a quick on demand backup with timeshift before each update just in case. But again, I only had to use that once a long time ago.
